doc/wallet-backup.md

```diff
@@ -1,91 +1,146 @@
-# Wallet Backup Instructions
+Wallet Backup Instructions
+==========================
```

Comment:
These changes should not be included; they are changing the documentation from Markdown to rST format (which is what the external repo required), but here we _want_ Markdown.